Resources/Prototypes/Entities/Structures/Specific/Anomaly/cores.yml

```diff
@@ -241,7 +241,7 @@
 
 - type: entity
   parent: BaseAnomalyInertCore
-  id: AnomalyCoreFleshInert
+  id: AnomalyCoreFleshInert # Can be microwaved to turn it into a massive piece of meat
```

Comment:
I think it would be a good idea to have a custom description for the flesh core that nudges the player into trying to microwave it. Otherwise, there is no realistic way to discover this interaction without meta knowledge

Reply:
Updated descriptions for both the regular and inert flesh cores!